package.json

```diff
@@ -70,7 +70,7 @@
     "pre-commit": "^1.0.0",
     "prettier": "^1.13.4",
     "standard-version": "^4.0.0",
-    "webpack": "^3.0.0"
+    "webpack": "next"
```

Comment:
Should be something like `^5.0.0-beta.15`

Reply:
I will fix it in other PRs